docs/04-api-reference/front-commerce-core/defineExtension.mdx

```diff
@@ -185,7 +185,7 @@ Example:
 
 Called when the server services are initialized with the following params:
 
-- [`Services`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/8373880e76c6d58449eeef623927d7e1d3458897/packages/core/services/Services.ts#L5-8) -
+- [`Services`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/core/services/Services.ts) -
```

Comment:
**issue:** The documentation shouldn't link to potentially unreleased code.

**suggestion:** use a specific commit (like previously) or a tag (won't be available until release).

Reply:
Fixed with your suggestion